core/cas-server-core-events-api/src/main/java/org/apereo/cas/support/events/listener/CasAuthenticationAuthenticationEventListener.java

```diff
@@ -110,4 +116,14 @@ public void handleCasRiskyAuthenticationDetectedEvent(final CasRiskyAuthenticati
         dto.setPrincipalId(event.getAuthentication().getPrincipal().getId());
         this.casEventRepository.save(dto);
     }
+
+    @Override
+    public void handleCasRequestSingleLogoutEvent(final CasRequestSingleLogoutEvent event) throws Throwable {
```

Comment:
I am not sure if I missed this: please add tests to ensure the event is actually received without mocks, and that logout is processed.

Reply:
You're fully right. I tested that the two events were published, but I never tested that the logout was actually performed. I will a test for that.